### PR TITLE
Add roles for organization membership

### PR DESCRIPTION
**Pull request by joshsmith**
Part of #26.

This adds missing roles and role validation with tests.

Also minor removal of an unused `import` in an unrelated file that was bothering me.

### CODE REVIEW

**Comment by joshsmith** [comment]
If someone could review this, that would be awesome.

File: web/models/organization_membership.ex [hunk 1]
```diff
@@ -18,23 +18,37 @@ defmodule CodeCorps.OrganizationMembership do
...
   def create_changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:member_id, :organization_id, :role])
-    |> validate_required([:member_id, :organization_id, :role])
+    |> changeset(params)
+    |> cast(params, [:member_id, :organization_id])
+    |> validate_required([:member_id, :organization_id])
     |> assoc_constraint(:member)
     |> assoc_constraint(:organization)
+    |> validate_inclusion(:role, roles)
```
Comment (npendery) [inline web/models/organization_membership.ex]: `validating _inclusion` twice, here and line 27 under changeset

**Comment by npendery** [comment]
Besides my one comment, 🐑 🇮🇹

File: web/models/organization_membership.ex (hunk 1 above)
Comment (joshsmith) [reply web/models/organization_membership.ex]: Good catch! Fixed now.

**Pull request merged**